CHANGELOG.md

```diff
@@ -1,5 +1,87 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.4.0
+
+<!-- generated comparing v5.3.1..master -->
+
+_Jan 31, 2022_
+
+A big thanks to the 22 contributors who made this release possible. Here are some highlights ✨:
+
+- 🛠 @goncalovf added an example project using MUI with Vite.js (#28241)
```

Comment:
I tried linking the [Vite.js example](https://github.com/mui-org/material-ui/tree/next/examples/vitejs) on this line but when going in it, it seems to be broken. Maybe we need to sort it out before releasing this as a highlight?!

Reply:
```suggestion
- 🛠 @goncalovf added an example project using [MUI with Vite.js](https://github.com/mui-org/material-ui/tree/master/examples/vitejs) (#28241)
```

Reply:
I've updated it :)